api/envoy/api/v2/route/route.proto

```diff
@@ -404,6 +404,9 @@ message CorsPolicy {
 
   // Specifies if CORS is enabled. Defaults to true. Only effective on route.
   google.protobuf.BoolValue enabled = 7;
+
+  // Specifies the root runtime key for flags related to this policy. Defaults to "".
+  string runtime_key = 9;
```

Comment:
Please have specific settings for each thing that you want to configure, and use `RuntimeFractionalPercent` for each one. This will allow it to work with both runtime and direct config with different incremental levels.